version.py

```diff
@@ -1 +1 @@
-__version__ = '1.4.2'
+__version__ = '1.0.0'
```

Comment:
I feel this is new module so new version should be fine. But I leave to maintainers for final call..